src/beakerlib.sh

```diff
@@ -31,6 +31,10 @@
 #   Boston, MA 02110-1301, USA.
 #
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Command aliases for compatibilities
+declare READLINK_CMD="readlink"
```

Comment:
There's a convention of prefixing internal variables with `__INTERNAL_`. Without it it is kinda expected the user might provide it through the env.
The current intention is rather internal variable, as I understood, so I would use the prefix.

Reply:
Roger that. Would fit even better like that after adding a Makefile interface for `sed`ing that.